install_sct.bat

```diff
@@ -0,0 +1,45 @@
+:: Usage: install_sct.bat <version>
+:: e.g.
+::        install_sct.bat 5.5
+
+:: Fetch version number (should be git branch, git tag, etc.)
+set git_ref=%1
+
+:: Save present working directory to return to later
+FOR /F "tokens=*" %%G IN ('cd') do (SET PWD=%%G)
+
+:: Go to user's home directory
+cd %HOMEPATH%

```

Comment:
For simplicity's sake (just to get us off the ground), I've hardcoded the script to install to the user's home directory.

While this isn't really kosher as far as Windows applications go, my thought process was that... because our end-goal is a pip-installable SCT, we're going to be discarding a lot of this fluff so that the user can install directly into a virtual environment. So, rather than go down the path of a Windows application, I'd rather be transparent about what's actually happening as far as virtual environments go.